sentry-android-core/src/test/java/io/sentry/android/core/ManifestMetadataReaderTest.kt

```diff
@@ -548,6 +548,66 @@ class ManifestMetadataReaderTest {
         assertTrue(fixture.options.isEnableNdk)
     }
 
+    @Test
+    fun `applyMetadata reads SDK name from metadata`() {
+        // Arrange
+        val expectedValue = "custom.sdk"
+
+        val bundle = bundleOf(ManifestMetadataReader.SDK_NAME to expectedValue)
+        val context = fixture.getContext(metaData = bundle)
+
+        // Act
+        ManifestMetadataReader.applyMetadata(context, fixture.options)
+
+        // Assert
+        assertEquals(expectedValue, fixture.options.sdkVersion?.name)
+    }
+
+    @Test
+    fun `applyMetadata does not override SDK name from options`() {
```

Comment:
the PR is failing because these tests are failing - I'm not sure they make sense, because the manifest metadata is actually overriding what was set in the Options ctor, right?

Reply:
yeah, they don't really make sense, I was copying it from one other option but I see it's only used there (sample rate). Removed the test cases